website/content/docs/setup/multicluster/index.md

```diff
@@ -1,8 +1,71 @@
 ---
 title: Multicluster Guide
 linktitle: Multicluster
-draft: true
+draft: false
 weight: 20
 ---
 
-TODO multicluster configuration options
\ No newline at end of file
+{{< name >}} is dynamic in its multi cluster capabilities in that it has abstractions to support multiple
+different cross-cluster orchestration mechanisms. The default mechanism that is described here is using [Razee](http://razee.io).
+
+
+## Multicluster operation with Razee
+
+Razee is a multi-cluster continuous delivery tool for Kubernetes that can deploy software on remote clusters and track
+the deployment status of such deployments. There are multiple ways to run Razee. The two desribed here are a vanilla open source deployment on your
+own Kubernetes or as a managed service from a cloud provider. The two different setups are described below.
+In general there is a need for:
+* Razee watch keeper (installed on all clusters)
+* Razee cluster subscription manager (installed on all clusters)
+* RazeeDash API (installed on controller cluster/as cloud service)
+
+### Installing Razee on Kubernetes
+
+An installation of the open source components is described [here](https://razee.io/#get-razee). 
+(TODO maybe add a description on how to install it including openshift routes etc)
+
+The credentials to access Razee have to be configured in a special secret in the `m4d-system` namespace.
+
+TODO Describe how to use Razee with API key and Github authentication once implemented.
+
+In order to configure {{< name >}} to use razee environment variables have to be configured for the 
+manager component.
+
+```
+envFrom:
+- secretRef:
+    name: razee-credentials
+- configMapRef:
+    name: m4d-config
+```
+
+### Installing using IBM Satellite Config
```

Comment:
Is this an alternative way to install Razee on a cluster? if so, can we emphasize, and add the context which is that IBM Cloud offers a aaS way to use Razee, and include prerequisite (i.e., at least need an IBM cloud account, use IAM keys, ...).